.travis.yml

```diff
@@ -10,6 +11,8 @@ node_js:
 env:
     global:
         - CXX=g++-4.8
+        - COZY_DESKTOP_DIR=tmp
+        - DEBUG=true
```

Comment:
I'm not sure it's a good idea to put `DEBUG=true` on travis. I think it's too verbose.